features/support/env.rb

```diff
@@ -32,7 +32,7 @@
 
 # Set up proxy server (used to manipulate HTTP headers etc since Selenium doesn't
 # support this)
-server = BrowserMob::Proxy::Server.new("./bin/browsermob-proxy")
+server = BrowserMob::Proxy::Server.new("./bin/browsermob-proxy", port: 3222)
```

Comment:
Can we make it randomly pick a port between a range? Might help if someone tries to run Smokey twice?